lib.Makefile

```diff
@@ -237,7 +237,8 @@ endif
 DOCKER_BUILD=docker buildx build --pull \
 	     --build-arg QEMU_IMAGE=$(CALICO_BUILD) \
 	     --build-arg UBI_IMAGE=$(UBI_IMAGE) \
-	     --build-arg GIT_VERSION=$(GIT_VERSION) $(TARGET_PLATFORM)
+	     --build-arg GIT_VERSION=$(GIT_VERSION) $(TARGET_PLATFORM) \
+	     --build-arg TARGETARCH=$(ARCH)
```

Comment:
nit, but to be consistent with `TARGET_PLATFORM` and our other args, let's call this `TARGET_ARCH`